frontend/src/components/Admin/TeamEvent/TeamEventDashboard.tsx

```diff
@@ -51,11 +52,58 @@ const TeamEventDashboard: React.FC = () => {
 
   if (isLoading) return <Loader active>Fetching team event data...</Loader>;
 
+  const handleExportToCsv = () => {
+    const csvData = allMembers.map((member) => {
+      const totalCredits = teamEvents.reduce(
+        (val, event) => val + calculateTotalCreditsForEvent(member, event),
+        0
+      );
+      const initiativeCredits = teamEvents.reduce(
+        (val, event) => val + calculateInitiativeCreditsForEvent(member, event),
+        0
+      );
+
+      const data = teamEvents.reduce(
+        (prev, event) => ({
+          ...prev,
+          [event.name]: calculateTotalCreditsForEvent(member, event)
+        }),
+        {
+          Name: `${member.firstName} ${member.lastName}`,
+          NetID: `${member.netid}`,
+          Total: totalCredits,
+          Initiative: initiativeCredits
```

Comment:
nit: camelCase the keys :)

Reply:
The keys show up exactly as the title of a column in the CSV file, which I feel like would look better starting with an uppercase letter since it's consistent with the names of the team events.